aiocoap/transports/ws.py

```diff
@@ -289,8 +289,24 @@ async def _connect_task(self, key: PoolKey):
 
             hostinfo_split = util.hostportsplit(key.hostinfo)
 
+            ws_scheme = {'coap+ws': 'ws', 'coaps+ws': 'wss'}[key.scheme]
+
+            if ws_scheme == 'ws' and ssl_context is not None:
+                raise ValueError("An SSL context was provided for a remote accessed via a plaintext websockets.")
+            if ws_scheme == 'wss':
+                if is_pyodide:
+                    if ssl_context is None:
```

Comment:
Are you sure it is `ssl_context is None` ? Reading the exception text below, it feels it should be `ssl_context is not None` as pyodide seems to only support default SSL context.